.circleci/config.yml

```diff
@@ -276,13 +276,13 @@ jobs:
       - run: yarn --cwd=dashboard run test --maxWorkers=4 --coverage
   test_pinniped_proxy:
     docker:
-      - image: circleci/rust
+      - image: circleci/rust:1.50
     steps:
       - checkout
       - run: cargo test --manifest-path ./cmd/pinniped-proxy/Cargo.toml
   test_chart_render:
     environment:
-      HELM_VERSION: "v3.1.0"
+      HELM_VERSION: "v3.5.2"
```

Comment:
this version is an old one in purpose. If we set this to the latest we won't know if we are relying in a version that's too recent. This should be the oldest version of Helm that we support.